app/src/common/shared/com/igalia/wolvic/ui/widgets/NavigationBarWidget.java

```diff
@@ -849,7 +870,8 @@ private void enterVRVideo(@VideoProjectionMenuWidget.VideoProjectionFlags int aP
         // the web content is already in fullscreen state.
         if (mFullScreenMedia == null)
             mFullScreenMedia = getSession().getActiveVideo();
-        assert mFullScreenMedia != null;
+
+        // mFullScreenMedia may still be null at this point
```

Comment:
Maybe add some more information, we do know now what happens but perhaps not in 5 months :)

Reply:
Done.